debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/transforms/UnwrapFromMongoDbEnvelope.java

```diff
@@ -38,6 +38,8 @@
  */
 public class UnwrapFromMongoDbEnvelope<R extends ConnectRecord<R>> implements Transformation<R> {
 
+    final String DEBEZIUM_OPERATION_HEADER_KEY = "application/debezium-operation";
```

Comment:
So I'm not sure whether there are any established conventions for naming header keys.

But if not, I'd suggest to go with `__debezium-operation`. The reason being that we use the `__debezium` prefix already in a few other places for injecting custom information, e.g. for the schema parameters conveying column source type info (`__debezium.source.column.type`) and the name of the heartbeat topic (`__debezium-heartbeat`). It'll still not be 100% consistent, but I think it's as good as it gets. @jpechane, any thoughts from your side?

Reply:
@gunnarmorling a ping! :)

Reply:
I think I went a bit off, I talked to a good Kafka person and I was told the HTTP is not really a reference there.
So I changed it to your suggestion :+1: